webplugin/js/app/labels/default-labels.js

```diff
@@ -161,5 +161,9 @@ Kommunicate.defaultLabels = {
 
         },
         'attachment': 'You have an attachment.'
-    }
+    },
+    'waiting.queue':{
+        'first.Part':'You are currently',
+        'waiting.queue.number':'5',
+        'last.part': ' in the waiting queue, our agents will get back you shortly.'},
```

Comment:
"waiting.queue.message" or "waiting.queue.heading"
The label name should be self-explanatory as the customers will use it.

NOTE: Please create the PR in docs repo also to add these labels.